### PR TITLE
Add email settings and redis db setting.

### PR DESCRIPTION
**Pull request by mzch**
Add email settings and redis db setting.

### CODE REVIEW

File: INSTALL.md [hunk 1]
```diff
@@ -277,6 +297,19 @@ docker run -p 9001:9000 --name clamav -d niilo/clamav-rest
 	        name: file-uploader
 	        maxAge: 15552000000 // 1000 * 60 * 60 * 24 * 180 (6 months)
 	    }
+    	email {
```
Comment (popey456963) [inline INSTALL.md:300]: weird indentation, there should be a ":" here.

File: config.default.hjson [hunk 2]
```diff
@@ -32,13 +32,27 @@
...
+        // url: redis://127.0.0.1:6379/0
         host: localhost
         port: 6379
+        db: 0
     }
     cookie: {
         name: file-uploader
         maxAge: 15552000000 // 1000 * 60 * 60 * 24 * 180 (6 months)
     }
+    email {
```
Comment (popey456963) [inline config.default.hjson:44]: no colon here

**Review by popey456963 — CHANGES_REQUESTED** [review]

**Comment by popey456963** [comment]
Apologies for the lengthy time reviewing this.  I've had some personal issues to deal with.

Just a few missing colons, otherwise looks great! 👍

File: INSTALL.md (hunk 1 above)
Comment (mzch) [reply INSTALL.md]: fixed.

File: config.default.hjson (hunk 2 above)
Comment (mzch) [reply config.default.hjson]: fixed

**Pull request merged**